File: tests/conformance/test-prompts-ruby/greet.rb
prompt description: "Greeting prompt",
  arguments: {
    name: "string",
    tone: { type: "string", optional: true, description: "formal or casual" }
  }

render do |args|
  name = args["name"] || "world"
  tone = args["tone"] || "casual"
  [{ "role" => "user", "content" => { "type" => "text", "text" => "Greet #{name} in a #{tone} tone" } }]
end

File: tests/conformance/test-prompts-python/greet.py
description = "Greeting prompt"
arguments = {
    "name": "string",
    "tone": {"type": "string", "optional": True, "description": "formal or casual"},
}

async def render(args):
    name = args.get("name", "world")
    tone = args.get("tone", "casual")
    return [
        {"role": "user", "content": {"type": "text", "text": f"Greet {name} in a {tone} tone"}},
    ]
